src/main/webapp/main-page.html

```diff
@@ -4,23 +4,25 @@
     <meta charset="UTF-8">
     <title>Main Page</title>
     <link rel="stylesheet" href="main-style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Nunito:ital,wght@1,300&display=swap" rel="stylesheet">
     <script src="./script.js"></script>
     <script src="./database-tables.js"></script>
+    <script src="table.js"></script>
     <script src="filters.js"></script>
-    <link rel="stylesheet" href="style.css">
   </head>
     
-  <body onload="mainLoad()">
+  <body onload="mainLoad(); createFilters()">
```

Comment:
Maybe instead of calling two methods here you can put the createFilters() method inside of the mainLoad method.

Reply:
Done!